src/store/feature-tags/index.js

```diff
@@ -0,0 +1,25 @@
+import { List, Map as $MAP } from 'immutable';
+import { RECEIVE_FEATURE_TOGGLE_TAGS, TAG_FEATURE_TOGGLE, UNTAG_FEATURE_TOGGLE } from './actions';
+
+function getInitState() {
+    return new List();
+}
+
+const featureTags = (state = getInitState(), action) => {
```

Comment:
In theory we could just store tags directly on the feature toggle in the feature toggle store, instead of maintaining a separate store for it.